platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/events/DetailedConsensusEvent.java

```diff
@@ -82,7 +116,21 @@ public void serialize(final SerializableDataOutputStream out) throws IOException
     public void deserialize(final SerializableDataInputStream in, final int version) throws IOException {
         this.gossipEvent = new GossipEvent();
         this.gossipEvent.deserialize(in, gossipEvent.getVersion());
-        consensusData = in.readSerializable(false, ConsensusData::new);
+
+        in.readInt(); // ConsensusData.version
+        in.readLong(); // ConsensusData.generation
+        in.readLong(); // ConsensusData.roundCreated
+        in.readBoolean(); // ConsensusData.stale
+        lastInRoundReceived = in.readBoolean();
+        final Instant consensusTimestamp = in.readInstant();
+        roundReceived = in.readLong();
+        final long consensusOrder = in.readLong();
+
+        final EventConsensusData eventConsensusData = EventConsensusData.newBuilder()
+                .consensusTimestamp(HapiUtils.asTimestamp(consensusTimestamp))
+                .consensusOrder(consensusOrder)
+                .build();
+        gossipEvent.setConsensusData(eventConsensusData);
```

Comment:
Does the byte format for this change? Not a huge deal if it does... but if so we should probably get signoff that it's ok to change byte format in CES without HIP.

Reply:
it does not, there is a unit test that checks it